packages/components/forma-36-react-datepicker/package.json

```diff
@@ -40,8 +41,6 @@
     "@types/react-dom": "^16.9.3",
```

Comment:
Should we remove the `@types` as well?

Reply:
Hmm I don't know, I will remove it and if it does not break we remove it for real hahahahah